berth_reservations/settings.py

```diff
@@ -47,6 +47,9 @@
     TOKEN_AUTH_AUTHSERVER_URL=(str, ""),
     TOKEN_AUTH_FIELD_FOR_CONSENTS=(str, "https://api.hel.fi/auth"),
     TOKEN_AUTH_REQUIRE_SCOPE_PREFIX=(bool, True),
+    VENE_PAYMENTS_ENABLED=(bool, True),
```

Comment:
respa had this setting because it is used by multiple cities, and some do not have payable resources. respa can be used without payments, but vene can't. I think we could skip this setting in our backend.

Reply:
True! I thought I had removed it everywhere but I missed this occurrence. 👍